installer/default_platform.conf

```diff
@@ -481,7 +481,7 @@ bootloader_menu_config()
     echo "Switch CPU vendor is: $CPUVENDOR"
     if echo "$CPUVENDOR" | grep -i 'Intel' >/dev/null 2>&1; then
         echo "Switch CPU cstates are: disabled"
-        CSTATES="intel_idle.max_cstate=0"
+        CSTATES="processor.max_cstate=0 intel_idle.max_cstate=0"
```

Comment:
Let's use 1 here. 
According to document, system will change it to 1 when user set it to 0.
To avoid confusion, let's use 1 here.